build-packages/magento-scripts/lib/tasks/file-system/create-php-storm-config.js

```diff
@@ -0,0 +1,71 @@
+const setConfigFile = require('../../util/set-config');
+const pathExists = require('../../util/path-exists');
+const path = require('path');
+const fs = require('fs');
+
+const createPhpStormConfig = () => ({
+    title: 'Setting PHPStorm config',
+    task: async ({ config: { phpStorm }, ports }) => {
+        const phpVersion = phpStorm.php.version;
```

Comment:
And if you change property name to `phpStorm.php.phpLanguageLevel` you will be able to get it nicely through destructuring assignment.
```js
const { phpLanguageLevel } = phpStorm.php;
```